docs/conf.py

```diff
@@ -1,4 +1,4 @@
-from crate.theme.rtd.conf.crash import *
+from crate.theme.rtd.conf.clients_crash import *
```

Comment:
Does this require a certain minimum version of the docs theme? If so, please add the minimum version to the requirements.

Reply:
good catch. I'll hold off until I do the release on the theme and then come back to this and take care of it

also, I need to take care of this: https://github.com/crate/crate/pull/9302#issuecomment-546934088

Reply:
okay fixed this now